db/seeds.rb

```diff
@@ -10,43 +10,15 @@
 require 'ffaker'
 expirations = ((Date.today - 7.days)..(Date.today + 6.months)).to_a
 
-10.times do
-  Passenger.create! name: FFaker::Name.name,
-                    address: FFaker::Address.street_address,
-                    email: FFaker::Internet.email,
-                    phone: FFaker::PhoneNumber.short_phone_number,
-                    wheelchair: false,
-                    active: true,
-                    permanent: false,
-                    expiration: expirations.sample
-  Passenger.create! name: FFaker::Name.name,
-                    address: FFaker::Address.street_address,
-                    email: FFaker::Internet.email,
-                    phone: FFaker::PhoneNumber.short_phone_number,
-                    wheelchair: false,
-                    active: true,
-                    permanent: true
+20.times do
```

Comment:
Not positive about being in equal frequency but yes, we're trying to cover all cases for expiration dates (at least, all cases for which we have color coding options).